articles/quickstart/spa/auth0-react/01.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to a React application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - react
+  - login
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+This guide demonstrates how to use the Auth0 React SDK with any new or existing React application.
+
+## Prerequisites
+
+- If you don't have an Auth0 account already, <a href="https://auth0.com/signup" data-amp-replace="CLIENT_ID" data-amp-addparams="anonId=CLIENT_ID(cid-scope-cookie-fallback-name)">sign up for a free Auth0 account</a>, and [create a tenant](https://auth0.com/docs/getting-started/create-tenant).
+
+- [Register your React application](https://auth0.com/docs/dashboard/guides/applications/register-app-spa) as a single-page application with Auth0.
+
+## Install the SDK
+
+Run the following command within your project directory to install the Auth0 React SDK:
+
+```bash
+npm install @auth0/auth0-react
+```
+
+> You may use `yarn` instead.
```

Comment:
This came up in some in some internal discussion last year during some research into quickstarts - the feeling was that this kind of thing was a bit redundant as, if you're a `yarn` user, then you already know how `yarn` can replace what `npm` is already doing. And if you're not already using `yarn`, then you're not likely to start using it just because we've mentioned it here.

Not sure what you think of that but just thought I'd point it out.